utils/python/CIME/case.py

```diff
@@ -77,6 +78,24 @@ def _get_env(self, short_name):
                   return env_file
           expect(False, "Could not find object for %s in case"%full_name)
 
+    def copy(self, newcasename, newcaseroot, newcimeroot=None, newsrcroot=None):
+        newcase = deepcopy(self)
```

Comment:
Let's first establish that we need to do a copy. Why not just create a new Case object that points to the clone?